.travis.yml

```diff
@@ -26,14 +26,14 @@ jobs:
     - stage: packaging
       python: "3.6"
       install:
-        - "pip install --upgrade twine readme-renderer[md]"
+        - "pip install --upgrade setuptools twine readme_renderer[md]"
       script:
         - "python setup.py sdist"
         - "twine check dist/*"
     - stage: coverage
       python: "3.6"
       install:
-        - "pip install --upgrade pip pipenv pytest-cov pytest-xdist pytest-timeout pytest"
+        - "pip install --upgrade pip pipenv pytest-cov pytest-timeout pytest"
```

Comment:
You removed pytest-xdist :smile: 
But I will investigate why multiprocess is still causing issues later.